packages/ember/src/components/form/radio-group.gts

```diff
@@ -0,0 +1,109 @@
+import { hash } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import BoundValue from './bound-value.ts';
+
+import type { Optional } from '../../types';
+import type { ComponentLike } from '@glint/template';
+
+export interface RadioGroupFieldSignature {
+  Element: HTMLDivElement;
+  Args: {
+    name: string;
+    basic?: boolean;
+    disabled?: boolean;
+    onChange?: (value: string, ...args: unknown[]) => void;
```

Comment:
Since this component extends the `BoundValue` class, you don't need to provide the type for `onChange`.
```suggestion
```